src/redisearch_rs/inverted_index/src/lib.rs

```diff
@@ -453,7 +454,7 @@ impl<'a> RSIndexResult<'a> {
             doc_id: 0,
             dmd: ptr::null(),
             field_mask: 0,
-            freq: 0,
+            freq: 1,
```

Comment:
This affects the frequency of everything since `default()` uses `virt()`

Reply:
Should be fine.
Where are we concerned about this value? For aggregation `RSAggregateResultIter`, the frequency is not used (and I'm not convinced that the summing we do even makes sense. Until recently, it wasn't even reset after each read).